jit-transfer-obj/src/main/java/org/dcsa/jit/enums/ModeOfTransport.java

```diff
@@ -0,0 +1,13 @@
+package org.dcsa.jit.enums;
```

Comment:
package should be org.dcsa.jit.transferobjects.enums

Reply:
Ah, right. Moved!